src/CalcViewModel/StandardCalculatorViewModel.cpp

```diff
@@ -222,10 +224,32 @@ void StandardCalculatorViewModel::SetParenthesisCount(_In_ const wstring& parent
     if (IsProgrammer || IsScientific)
     {
         OpenParenthesisCount = ref new String(parenthesisCount.c_str());
-        RaisePropertyChanged("LeftParenthesisAutomationName");
     }
 }
 
+void StandardCalculatorViewModel::OpenParenthesisCountNarratorAnnouncment()
+{
+    String^ parenthesisCount = ((m_OpenParenthesisCount == nullptr) ? "0" : m_OpenParenthesisCount);
+    wstring localizedParenthesisCount = std::wstring(parenthesisCount->Data());
```

Comment:
Don't need explicit `std` namespace in cpps.  Can use `wstring localizedParenthesisCount = parenthesisCount->Data()`